src/appengine/handlers/cron/load_bigquery_stats.py

```diff
@@ -140,21 +119,24 @@ def _load_data(self, bigquery, fuzzer):
       else:
         schema = kind.SCHEMA
 
-      self._update_schema_if_needed(bigquery, dataset_id, table_id, schema)
-
       gcs_path = fuzzer_stats.get_gcs_stats_path(kind_name, fuzzer, timestamp)
+      load = {
+          'destinationTable': {
+              'projectId': project_id,
+              'tableId': table_id + '$' + date_string,
+              'datasetId': dataset_id,
+          },
+          'schemaUpdateOptions': ['ALLOW_FIELD_ADDITION',],
+          'sourceFormat': 'NEWLINE_DELIMITED_JSON',
+          'sourceUris': ['gs:/' + gcs_path + '*.json'],
+          'writeDisposition': 'WRITE_TRUNCATE',
```

Comment:
WRITE_TRUNCATE is the one which allows passed schema to overwrite field type over what is currently in table.